README.md

```diff
@@ -81,7 +81,7 @@ Execute the command with your venv (or virtualenv) environment.
 
 ### Option: from
 
-By default, this tool finds the license from package Metadata (`--from=meta`). However, depending on the type of package, it does not declare a license only in the Classifiers.
+By default, this tool finds the license from [Trove Classifiers](https://pypi.org/classifiers/) or package Metadata. Some Python packages declare their license only in Trove Classifiers.
 
```

Comment:
Since pip-licenses 3.0, mixed mode is the standard behavior. 

This is an acceptance of Proposal #60, because mixed mode is more convenient for many users.